test/cli/cli_test.hpp

```diff
@@ -59,9 +57,9 @@ struct cli_test : public ::testing::Test
     {
         // Assemble the directory name.
         ::testing::TestInfo const * const info = ::testing::UnitTest::GetInstance()->current_test_info();
-        std::filesystem::path const test_dir{std::string{OUTPUTDIR} +
-                                             std::string{info->test_case_name()} +
-                                             std::string{"."} +
+        std::filesystem::path const test_dir{std::string{OUTPUTDIR} +              //
+                                             std::string{info->test_case_name()} + //
+                                             std::string{"."} +                    //
```

Comment:
Why the "slashes"? Is this clang formatting?

Reply:
Yes, it prevents clang-format from putting everything on two lines. Alternatively, I can wrap it in a `// clang-format off/on`?

Reply:
that would be even more annoying xD